src/xcode/ENA/ENA/Source/Views/BaseElements/ENATanInput.swift

```diff
@@ -103,6 +103,8 @@ class ENATanInput: UIControl, UIKeyInput {
 				label.textColor = labelTextColor
 				label.font = font
 				label.text = "-"
+				label.adjustsFontSizeToFitWidth = true
+				label.minimumScaleFactor = 0.2
 				stackView.addArrangedSubview(label)
```

Comment:
Okay for now. Will change the layout on the input field in due course as discussed with design team.

Reply:
Alright, thanks for the info!